migrations/20240826202953_feedback_v2.up.sql

```diff
@@ -0,0 +1,8 @@
+-- Add up migration script here
+
+CREATE TYPE feedback_type AS ENUM
```

Comment:
First obvious nitpick here is just moving this migration into the first one. No reason to make 2 migrations since the first one didn't reach prod.

Reply:
Alright, will fix right now.